DeviceSDKAndroid/src/main/java/org/astarteplatform/devicesdk/android/AstarteAndroidCryptoStore.java

```diff
@@ -13,6 +13,7 @@
 import java.security.PublicKey;
 import java.security.cert.Certificate;
 import java.security.cert.CertificateException;
+import java.util.logging.Logger;
```

Comment:
For Android package use Android API for logging
```suggestion
import android.util.Log;
```